js/guides.js

```diff
@@ -133,7 +133,18 @@ function postProcessCodeBlocks() {
     //  3) There is exactly 1 small set of languages to choose from. This does not allow for multiple language preferences. For example, users cannot prefer both Kotlin and ZSH.
     //  4) Only 1 sample of each language can exist in the same collection.
 
-    var preferredBuildScriptLanguage = window.localStorage.getItem("preferred-gradle-dsl") || "groovy";
+    var GRADLE_DSLs = ["groovy", "kotlin"];
```

Comment:
@acourtneybrown these next 20 lines are the only real changes here. Everything else is formatting.